apps/codeforafrica/src/lib/data/local/index.js

```diff
@@ -0,0 +1,21 @@
+import { getPageStaticProps } from "@/codeforafrica/lib";
+import { getPageProps } from "@/codeforafrica/lib/data/common";
+import { api } from "@/codeforafrica/lib/data/rest";
+
+export async function getPageServerSideProps(context) {
+  const props = await getPageProps(api, context);
+  if (!props) {
+    return { notFound: true };
+  }
+  // TODO Remove static props
+  const { props: staticProps } = await getPageStaticProps({ slug: "/" });
```

Comment:
???

Reply:
@kilemensi this is just a placeholder to populate home page with static props from  ghost-cms

Reply:
Lets move it to an issue then @koechkevin